src/software/new_geom/angle.h

```diff
@@ -103,6 +103,16 @@ class Angle final
      */
     constexpr double toDegrees() const;
 
+    /**
+     * Computes the modulus of a division between this angle and another
+     * angle.
+     *
+     * @param divisor the divisor.
+     *
+     * @return the modulus of this Angle ÷ divisor.
+     */
+    constexpr Angle mod(Angle divisor) const;
```

Comment:
Do we have tests for this?

Reply:
Added tests in the latest update

Reply:
Got lazy towards the end of the big pr but yeah, added tests in latest update